src/OpenTelemetry.Api/Context/Propagation/TraceContextPropagator.cs

```diff
@@ -259,7 +259,7 @@ private static byte HexCharToByte(char c)
                 return Convert.ToByte(c);
             }
 
-            throw new ArgumentOutOfRangeException(nameof(c), $"Invalid character: {c}.");
+            throw new ArgumentOutOfRangeException(nameof(c), c, $"Invalid character: {c}.");
```

Comment:
Consider putting the ordinal of the character as well, in case the character is non-printable.

Reply:
Thanks for the review. I've concisely added this on another commit. Since the branch has been merged would you recommend creating another PR to get this included, or is there a way to do it from here?
https://github.com/Austin-Tan/opentelemetry-dotnet/commit/a4205d3f57cf5bc05e214013b257a9e945f29f68

Reply:
Please send a separate PR, thanks.